include/zfs_fletcher.h

```diff
@@ -88,7 +88,7 @@ typedef struct zfs_fletcher_avx512 {
 } zfs_fletcher_avx512_t;
 
 typedef struct zfs_fletcher_aarch64_neon {
-	uint64_t v[2] __attribute__((aligned(16)));
```

Comment:
Why do we drop the aligned attribute?

Reply:
I have a patch doing that pending anyway, because we're lying when we claim that.

Reply:
Good point.

Reply:
Not quite sure here, since I re-write the Neon fletcher4 according to the superscalar4, so I need to align with the superscalar4 data structure..